content/nginxaas-azure/quickstart/disaster-recovery.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Disaster Recovery
+weight: 800
+toc: true
+url: /nginxaas/azure/quickstart/disaster-recovery/
+type:
+- how-to
+---
+
+## Disaster Recovery Configuration for F5 NGINX as a Service for Azure Deployments
```

Comment:
```suggestion

```

Reply:
We generally don't start docs with a heading that is an extended version of the title. I will look like a duplicate of the `title:` above in the page (almost same size and font)

![image](https://github.com/user-attachments/assets/7e803ea9-5b17-4bc8-9db1-92517e32208b)